### PR TITLE
Add Portland 2025 landing page

### PR DESCRIPTION
**Pull request by ericholscher**
This does the initial bits of:

* Copying yearly CSS & templates
* Copying over content files
* Updating welcome post: https://writethedocs-www--2163.org.readthedocs.build/conf/portland/2025/news/welcome/
* Updating config

----
📚 Documentation preview 📚: https://writethedocs-www--2163.org.readthedocs.build/

### CODE REVIEW

**Comment by swapnilogale** [comment]
We talking 2025 already! 😱

File: docs/_data/portland-2025-config.yaml [hunk 1]
```diff
@@ -0,0 +1,286 @@
...
+    price: $2,500
+  publisher:
+    price: $5,000
+  patron:
+    price: $8,500
+  keystone:
+    price: $14,000
+
+grants:
+  url: 'https://docs.google.com/forms/d/e/1FAIpQLScm8-8RLZYSxLl3a5Ttz5qAB0dXCn5E2IUL2U23jz2kSa3vmA/viewform'
```
Comment (mxsasha) [inline docs/_data/portland-2025-config.yaml]: We should comment this out as long as it links to the 2024 form.

File: docs/_data/portland-2025-config.yaml [hunk 2]
```diff
@@ -0,0 +1,286 @@
...
+
+cfp:
+  url: https://pretalx.com/wtd-portland-2025/cfp  # Open November 6 (Mon) -> 9 weeks to submit
+  ends: "January 15, 2025"                        # (Mon) 1 week, to review. It'll take a little longer to announce
+  notification: "January 30, 2025"                # (Mon) 8 weeks to write and prepare
+  slides_by: "April 5, 2025"                      # (Mon) 1 week before
+  preview: "https://writethedocs-www--2098.org.readthedocs.build/conf/portland/2025/schedule/"                                 # Preview URL TODO
+
+unconf:
+  url: http://bit.ly/wtd-pdx-2025-ro
```
Comment (mxsasha) [inline docs/_data/portland-2025-config.yaml]: Same here, if we don't have it yet, it should not be in the yaml. I think the templates are written so you can just comment out the whole unconf block.

File: docs/conf/portland/2025/news/welcome.rst [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+Tickets will go on sale in {{ date.tickets_live }}.
+The ticket pricing will be as follows:
+
+* **Student or Unemployed Tickets**: *{{tickets.student.price}}*
+* **Independent Tickets**: *{{tickets.independent.price}}*
+* **Corporate Tickets**: *{{tickets.corporate.price}}*
+
+We will be also be running our Opportunity Grants program again this year for those who need financial assistance to attend.
+We also have many opportunities for volunteers who want to help make the conference happen and get a free ticket that way.
+
```
Comment (mxsasha) [inline docs/conf/portland/2025/news/welcome.rst:40]:
```suggestion
Note that you are not required to do volunteer tasks at the conference to be able to apply for an Opportunity Grant.

```
This is a really common confusion, and I think we need to stress it when these sentences are close to each other.

**Review by mxsasha — COMMENTED** [review]
Did not read all 6000 lines of course, just news post and config. Left a few small notes.

File: docs/_data/portland-2025-config.yaml (hunk 1 above)
Comment (plaindocs) [reply docs/_data/portland-2025-config.yaml]: Yeah, in some places the templates don't like empty strings, but it's warning only.

File: docs/_data/portland-2025-config.yaml [hunk 4]
```diff
@@ -0,0 +1,286 @@
...
+  unconfroom: Library & Astoria room (1st floor)
+  projector_ratio: "16:9"
+  job_fair_room: Sponsor area
+
+cfp:
+  url: https://pretalx.com/wtd-portland-2025/cfp  # Open November 6 (Mon) -> 9 weeks to submit
+  ends: "January 15, 2025"                        # (Mon) 1 week, to review. It'll take a little longer to announce
+  notification: "January 30, 2025"                # (Mon) 8 weeks to write and prepare
+  slides_by: "April 5, 2025"                      # (Mon) 1 week before
+  preview: "https://writethedocs-www--2098.org.readthedocs.build/conf/portland/2025/schedule/"                                 # Preview URL TODO
```
Comment (plaindocs) [inline docs/_data/portland-2025-config.yaml]:
```suggestion
cfp:
  url: https://pretalx.com/wtd-portland-2025/cfp  # Open October 6 (Mon) -> 9 weeks to submit
  ends: "January 21, 2025"                        # (Mon) 1 week, to review. It'll take a little longer to announce
  notification: "February 04, 2025"                # (Mon) 8 weeks to write and prepare
  slides_by: "April 28, 2025"                      # (Mon) 1 week before
  preview: "TODO"                                 # Preview URL TODO
```

How do we feel about this? Too tight? Between notification and conference? Too long in the CFP?

File: docs/conf/portland/2025/news/welcome.rst [hunk 5]
```diff
@@ -0,0 +1,66 @@
+:template: {{year}}/generic.html
+
+.. post:: June 14, 2024
+   :tags: {{shortcode}}-{{year}}
+
+Announcing Write the Docs {{ city }} {{year}}
+===========================================
+
+We're excited to announce the dates for our Portland 2025 conference!
```
Comment (plaindocs) [inline docs/conf/portland/2025/news/welcome.rst]:
```suggestion
We're excited to already be able to announce the dates for our Portland 2025 conference! Block those calendars off now!
```

File: docs/conf/portland/2025/news/welcome.rst [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+**Write the Docs {{ city }} {{ year }} will be on {{ date.short }}**
+
+We'll be back at Revolution Hall for another year,
+full of great talks, networking, and community building.
+
+Call for Proposals
+------------------
+
+Every year, Write the Docs invites people from all across our community to come up on stage to share their insights and experience.
+**We plan to launch our CFP in November**,
```
Comment (plaindocs) [inline docs/conf/portland/2025/news/welcome.rst]: Pending CFP edits

**Review by plaindocs — APPROVED** [review]
Mostly reviewed config / welcome post, which look good.

Needs a bullet in the confeence includes page for https://writethedocs-www--2163.org.readthedocs.build/

File: docs/conf/portland/2025/news/welcome.rst (hunk 6 above)
Comment (ericholscher) [reply docs/conf/portland/2025/news/welcome.rst]: I updated this to not mention a specific date.

**Pull request merged**